src/sentry/conf/server.py

```diff
@@ -1092,6 +1093,15 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
         ),
         "options": {"expires": 60 * 60 * 3},
     },
+    "schedule-daily-organization-reports": {
+        "task": "sentry.tasks.summaries.daily_summary.schedule_organizations",
+        "schedule": crontab(
+            minute=0,
+            hour="*/1",  # Run every hour
```

Comment:
I was kinda debating changing this to only run at 4PST for the MVP since it's internal only, though that'd only hit the local people.